forte/processors/base/query_processor.py

```diff
@@ -65,5 +65,6 @@ def _process_query(self, input_pack: PackType) \
 
     def _process(self, input_pack: PackType):
         query_pack, query_value = self._process_query(input_pack)
-        query = Query(pack=query_pack, value=query_value)
+        query = Query(pack=query_pack)
```

Comment:
The design of the ``_process_query`` seem to be a little difficult, especially on returning `input_pack`.

Reply:
This does not affect the functionality and high-level design. I will create a separate issue for this.